sdk/eventhub/Azure.Messaging.EventHubs/tests/Producer/EventHubBufferedProducerClientOptionsTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Messaging.EventHubs.Core;
+using Azure.Messaging.EventHubs.Producer;
+using NUnit.Framework;
+
+namespace Azure.Messaging.EventHubs.Tests
+{
+    /// <summary>
+    ///   The suite of tests for the <see cref="EventHubBufferedProducerClientOptions"/>
+    ///   class.
+    /// </summary>
+    ///
+    [TestFixture]
+    internal class EventHubBufferedProducerClientOptionsTests
```

Comment:
```suggestion
    public class EventHubBufferedProducerClientOptionsTests
```

Reply:
We can leave the test classes `public`.    They won't get published for a package release.